### PR TITLE
Default registry, cafile

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:

Sets default npm registry and sets cafile to default value (undefined), for this project only.

### Resolves:

Developers may face issues installing behind corporate proxies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "packages/@okta/*"
   ],
   "scripts": {
+    "preinstall": "npm config delete cafile",
```
Comment (swiftone) [inline package.json]: What impact does this have to someone that has an existing .npmrc config outside of this module?   (does it change that .npmrc file?)

Comment (aarongranick-okta) [reply package.json]: It does not change any files. The setting seems to remain in effect (globally) possibly until I restart my machine. `npm config get cafile`

**Comment by nbarbettini** [comment]
We need someone with strap to test this and make sure it doesn't mess anything up.

File: package.json (hunk 1 above)
Comment (aarongranick-okta) [reply package.json]: Wrong about that. It definitely removes it from the file.

Comment (swiftone) [reply package.json]: In that case, I'm definitely not in favor of this change in a repo that non-okta users can clone.  Thanks for checking!

**Comment by aarongranick-okta** [comment]
I have not found a way of overriding `cafile` without causing destruction to `~/.npmrc`

**Pull request closed without merging**